aten/src/ATen/templates/Tensor.h

```diff
@@ -87,6 +87,9 @@ struct Tensor : public detail::TensorBase {
     return pImpl->unsafeGetTH(retain);
   }
 
+  // Purposely not defined here to avoid inlining
+  void print() const;
```

Comment:
nit: I'd rename that to `debugPrint` to make it clear that it's not something you should be using in the code

Reply:
I'll change that in another commit (there are a few other places where we call it `print`, so we would want to rename all of them if we change this.)